packages/bar/src/BarTotals.tsx

```diff
@@ -0,0 +1,29 @@
+import { useTheme } from '@nivo/core'
+import { BarTotalsData } from './compute/totals'
+
+interface Props {
+    barTotals: BarTotalsData[]
+}
+
+export const BarTotals = ({ barTotals }: Props) => {
```

Comment:
Sorry, I didn't think about it when first reviewing the PR, but how does this looks with animated charts? all other components support transitions, so it could be weird if we just have this not animated.

Reply:
I just added it following the implementation structure for the bars itself.

Reply:
Thank you.